tests/hub-local/test_hosted_instance_deletion_gate.py

```diff
@@ -0,0 +1,40 @@
+from laminhub_rest.test.utils.instance import (
+    create_hosted_test_instance,
+    delete_test_instance,
+)
+from lamindb_setup.core.upath import create_path
+from lamindb_setup.core._settings_storage import TOUCH_FILE_PATH
+from lamindb_setup.core._hub_core import delete_instance
+from lamindb_setup import settings
+import pytest
+
+
+def test_hosted_instance_deletion_gate(run_id, s3_bucket):
+    test_instance = create_hosted_test_instance(
+        f"test_instance_{run_id}", s3_bucket.name
+    )
+
+    # Make sure 0-byte file is touched upon storage initialization
+    path = create_path(
+        test_instance.storage_root, access_token=settings.user.access_token
```

Comment:
Do not pass `settings.user.access_token` manually.

Reply:
This will become irrelevant if we just leverage `ssetings.root`?

Reply:
No, it should not be done in general.

Reply:
You need to pass it explicitly only when you need a token different from the token of a current user.